src/once_cell.rs

```diff
@@ -101,3 +110,43 @@ impl<T> GILOnceCell<T> {
         Ok(())
     }
 }
+
+/// Interns `text` as a Python string and stores a reference to it in static storage.
+///
+/// A reference to the same Python string is returned on each invocation.
+///
+/// # Example: Using `intern!` to avoid needlessly recreating the same Python string
+///
+/// ```
```

Comment:
This needs a test that actually runs the code.

Reply:
Yes, this is why I went with the executable doctest in the beginning. Will add a unit test which will also yield coverage for this.

Reply:
Unless you intended to do this all along and I misunderstood 👍

Reply:
> Unresolving this because I do want this test to actually run its code and have its results checked.

Please check commit 30d414fd49f9e37c8d74963f66085613e6ee82fe (the last of the PR), it adds a unit test called `test_intern`. As written above, I chose a library-level unit test instead of an executable doc test but cargo-llvm-cov is not yet able to include doc tests in the coverage computation.

Reply:
> Unresolving this because I do want this test to actually run its code and have its results checked.

Ah, I think I get it now: You do not feat that the `intern!` macro does not work, but rather that the example itself is wrong in some subtle way. Will execute the function...